provider/src/main/kotlin/org/gradle/kotlin/dsl/resolver/SourcePathProvider.kt

```diff
@@ -54,7 +53,6 @@ object SourcePathProvider {
     private
     fun subDirsOf(dir: File): Collection<File> =
         if (dir.isDirectory)
-            dir.listFiles().filter { it.isDirectory }
-        else
+            dir.listFiles().filter { it.isDirectory } else
```

Comment:
❌  This change looks off to me

Reply:
This is from the IntelliJ settings that ktlint suggests you use in your IDE but is not required.
It may be in a future release of ktlint.